src/test/ui/privacy/privacy2.stderr

```diff
@@ -11,6 +11,10 @@ LL |     use bar::glob::foo;
    |                    ^^^
 
 error: requires `sized` lang_item
+  --> $DIR/privacy2.rs:14:14
+   |
+LL | pub fn foo() {}
+   |              ^
```

Comment:
@jonas-schievink This and privacy3 are expected?

Reply:
Probably, but yeah it's not a very good error span. Feel free to remove the spans for the `Sized` lang item, the compiler needs that lang item for all sorts of checks that don't really correspond to any language features.

Reply:
Hmm, okay, it may cause confusing so I'd like to remove their spans.